frontend/main/src/layout/Footer.tsx

```diff
@@ -111,7 +111,7 @@ export default function Footer({
           {/* HELPFUL LINKS */}
           <section className="grid grid-cols-1 lg:justify-self-end 2xl:justify-self-center">
             <h4 className="underline whitespace-nowrap">Helpful Links</h4>
-            <div className="grid w-64 grid-cols-2 gap-4">
+            <div className="grid grid-cols-1 gap-2 mt-4">
```

Comment:
I don't think we will want this, there are some more links that we will need to add and it will look weird
![image](https://user-images.githubusercontent.com/3102249/141191775-c125c892-ceeb-4088-8231-36933bd53a28.png)

Reply:
It just looked weird with 3 there, we can add the grid-cols-2 back when we add more links.